src/Resources/config/services.php

```diff
@@ -15,5 +15,6 @@
                 abstract_arg('API Key'),
                 abstract_arg('Organisation'),
             ])
-        ->alias('openai', Client::class);
+        ->alias('openai', Client::class)
+        ->alias(Client::class);
```

Comment:
I don't have auto wiring issue in my project. The service is already named with the class name.

And the 2nd argument of alias is required.

Reply:
Yup, and the 2nd argument can not be the same class name. So the PR was dead before u even reviewed it ^^